scripts/webpack.js

```diff
@@ -20,13 +20,11 @@ const rules = () => ({
       },
     },
   }),
-  css: () => [
-    {
-      test: /\.css$/,
-      include: ['ol', 'redux-notifications', 'react-datetime', 'codemirror'].map(moduleNameToPath),
-      use: ['to-string-loader', 'css-loader'],
-    },
-  ],
+  css: () => ({
```

Comment:
Any reason for this change?

Reply:
Not that I know of; I assumed this was done by the linter or `yarn format`